makelib/helm.mk

```diff
@@ -113,6 +118,16 @@ helm.promote: $(HELM_HOME)
 	@rm -fr $(HELM_TEMP)
 	@$(OK) promoting helm charts
 
+helm.museum: $(HELM_OUTPUT_DIR)/$(1)-$(HELM_CHART_VERSION).tgz
+	ifneq ($(MUSEUM_URL),)
+		@$(INFO) pushing helm charts to chart museum
```

Comment:
want to include the museum URL in this INFO statement?

Reply:
done